Automattic/social-logos
# Update svg attrs to camel case to avoid react dom warning

With React 15, SVG is now supported but attributes must be 🐫  cased. This will throw a warning otherwise. https://facebook.github.io/react/docs/dom-elements.html#all-supported-svg-attributes

We accidentally fixed this upstream in Calypso first:  https://github.com/Automattic/wp-calypso/pull/8590

This PR updates the grunt task to transform svg attributes from kabob-case to camelCase. 

## Testing Instructions
- `npm install`
- Install system dependent font deps https://github.com/sapegin/grunt-webfont
- `$(npm bin)/grunt --verbose`
- generated output should be the same

cc @drw158 @folletto @obenland
